commands/Fun/choice.js

```diff
@@ -0,0 +1,26 @@
+exports.run = (client, msg, choices) => {

```

Comment:
How I understood it is that the user is going to type in

```
!choice Option 1|Option 2|Option3
```

and the framework passes in

```js
['Option 1', 'Option 2', 'Option 3']
```

as the `choices` arg. Wouldn't that make it just take in the first option only?